railties/test/application/configuration_test.rb

```diff
@@ -2422,6 +2422,20 @@ def index
       assert_equal 1234, ActiveSupport.test_parallelization_threshold
     end
 
+    test "ActiveSupport.use_rfc4122_namespaced_uuids is enabled by default for new apps" do
+      app "development"
+
+      assert_equal true, ActiveSupport.use_rfc4122_namespaced_uuids
+    end
+
+    test "ActiveSupport.use_rfc4122_namespaced_uuids is disabled by default for upgraded apps" do
+      remove_from_config '.*config\.load_defaults.*\n'
```

Comment:
It looks like this kind of test is used earlier in this file, but I find it weird that this test removes the entire `config.load_defaults` as that is not particular to the feature we are testing here.

Would it make more sense explicitly remove (or change) just this setting in a generated app?